#! /bin/sh -e
## makeself.dpatch
## Bartosz Fenski <fenio@debian.org>

patch_opts="${patch_opts:--f --no-backup-if-mismatch ${2:+-d $2}}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
	-patch) patch $patch_opts -p1 < $0;;
	-unpatch) patch $patch_opts -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac
 
exit 0

@DPATCH@
--- makeself-2.1.2.orig/makeself.sh     2003-07-05 03:38:05.000000000 +0200
+++ makeself-2.1.2/makeself.sh  2004-03-24 23:50:04.000000000 +0100
@@ -98,7 +98,7 @@
 APPEND=n
 COPY=none
 TAR_ARGS=cvf
-HEADER=`dirname $0`/makeself-header.sh
+HEADER=/usr/share/makeself/makeself-header
 
 # LSM file stuff
 LSM_CMD="echo No LSM. >> \"\$archname\""

